how-to/run-kata-with-k8s.md

```diff
@@ -114,7 +114,7 @@ default_workload_trust = "untrusted"
 To enable networking for the workloads run by Kata, CRI-O needs to be configured to
 manage network namespaces, by setting the following key to `true`:
 ```toml
-manage_network_ns_lifecycle = true
+manage_ns_lifecycle = true
```

Comment:
Until we start forking the docs repository (https://github.com/kata-containers/documentation/issues/246), I think we should specify both versions of this option along with the version ranges of crio they apply to. This is a pita but if a user running a stable release looks at this doc once it lands, they will (potentially) be seeing the wrong information (if they are still using crio 1.16.1).

Also, you might want to add a link to https://github.com/kata-containers/runtime/pull/2458 in the commit since that PR is what triggered this one from being raised I think?

/cc @mvincerx.

Reply:
Ok thanks for the feedback, I specified both.